client/src/components/settings/sound.tsx

```diff
@@ -28,6 +49,20 @@ export default function SoundSettings({
           toggleSoundMode(sound ? false : true);
         }}
       />
+      <label htmlFor='volumeslider'>
+        {t('settings.sound-volume')}{' '}
+        <span aria-hidden='true'>{volumeDisplay}</span>
+      </label>
+      <input
+        type='range'
+        min='10'
```

Comment:
Not sure if this would throw off the method for calculating decibels, but it might be better for `min` to be 0 here:

```suggestion
        min='0'
```

This should make the range more evenly distributed, and should make sure the slider is centered when someone sets the volume to 50.

Reply:
I would rather leave it as it is, if the volume is set to 0 I think we want to user to turn off campfire mode.

They would not know that it is turned on.

Reply:
Sure, that makes sense. I'll do another quick round of testing and mark this as approved.